sqlalchemy-stubs/dialects/postgresql/ext.pyi

```diff
@@ -0,0 +1,28 @@
+from ...sql import expression, functions, visitors
+from ...sql.schema import ColumnCollectionConstraint
+from .array import ARRAY
+from typing import Any, Optional, Union, TypeVar, Dict, Tuple
+
+_T = TypeVar('_T')
+_AOB = TypeVar('_AOB', bound=aggregate_order_by)
+
+class aggregate_order_by(expression.ColumnElement[Any]):
+    __visit_name__: str = ...
+    target: expression.ColumnElement[Any] = ...
+    order_by: Any = ...
+    def __init__(self, target: expression.ColumnElement[Any], *order_by: expression.ClauseElement) -> None: ...
+    def self_group(self: _AOB, against: Optional[Any] = ...) -> _AOB: ...
+    def get_children(self, **kwargs: Any) -> Tuple[expression.ColumnElement[Any], Any]: ...
+
+_EC = TypeVar('_EC', bound=ExcludeConstraint)
+
+class ExcludeConstraint(ColumnCollectionConstraint):
+    __visit_name__: str = ...
+    where: Optional[visitors.Visitable] = ...
+    operators: Dict[str, Any] = ...
+    using: str = ...
+    def __init__(self, *elements: Any, name: str = ..., deferrable: bool = ..., initially: str = ..., using: str = ...,
+                 where: Optional[Union[str, bool, visitors.Visitable]] = ...) -> None: ...
+    def copy(self: _EC, **kw: Any) -> _EC: ...
+
+def array_agg(*arg: Any, **kw: Any) -> functions.array_agg[_T]: ...
```

Comment:
Why the type variable only appears in the return type? I think this doesn't make much sense.

Reply:
Yeah, this typing is wrong. I'll fix it.